webapp/src/components/common/DataGrid.tsx

```diff
@@ -38,9 +38,12 @@ type RowMarkers =
 type RowMarkersOptions = Exclude<RowMarkers, string>;
```

Comment:
You only need to do this:

```typescript
export interface DataGridProps extends Omit<DataEditorProps, "rowMarkers" | "gridSelection"> {
  rowMarkers?: RowMarkers;
  enableColumnResize?: boolean;
}

// ...

const handleGridSelectionChange = (newSelection: GridSelection) => {      
   // ...

   setSelection(newSelection);

   onGridSelectionChange?.(newSelection);
  };
```

You doesn't need to add `gridSelection` in props.

Reply:
done thanks